.evergreen.yml

```diff
@@ -67,7 +67,16 @@ functions:
       params:
         working_dir: spruce/evergreen
         command: make configure-mongod
-
+    - command: shell.exec
+      type: setup
+      params:
+        shell: bash
+        script: |
+          ${PREPARE_SHELL}
+          curl -LO https://fastdl.mongodb.org/tools/db/mongodb-database-tools-ubuntu2204-x86_64-100.7.5.tgz;
+          tar -zxvf mongodb-database-tools-ubuntu2204-x86_64-100.7.5.tgz
+          cd mongodb-database-tools-ubuntu2204-x86_64-100.7.5/bin
```

Comment:
This would surprise me if this isn't already downloaded when we download MongoDB they are usually in the same directory.

Could we also use the same process we use for installing MongoDB elsewhere in this script? e.g. Put these variables into expansions and use a make file.